common/changes/@reshuffle/db/grammar-suggestions_2019-09-25-06-34.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
You could avoid this change entry by hitting ENTER instead of entering text when `rush change` asks.  This is change does not need a new version number AFAICS.

Reply:
Note that a `none` change would only be published with later patches, so anyone using the package with vscode will not see the documentation before the next publish, so it's a tradeoff between how often we want to update.